skore/src/skore/sklearn/_cross_validation/metrics_accessor.py

```diff
@@ -168,14 +177,27 @@ def _compute_metric_scores(
                 keys=[f"Split #{i}" for i in range(len(results))],
             )
             results = results.swaplevel(0, 1, axis=1)
+
+            # pop the favorability column if it exist to:
+            # - no use it in the aggregate operation
+            # - later to only report a single column and not by split columns
```

Comment:
```suggestion
            # Pop the favorability column if it exists, to:
            # - not use it in the aggregate operation
            # - later to only report a single column and not by split columns
```
What is meant by the second bullet point?